models/recommendation.js

```diff
@@ -43,4 +61,17 @@ const Recommendation = sequelize().define('Recommendation', {
   timestamps: false
 })
 
+Recommendation.prototype.__defineGetter__('category', function () {
+  const category = last(this.categories) || DEFAULT_CATEGORY
+  return titleCaps(category.replace(/-/gi, ' '))
+})
+
+Recommendation.prototype.findRecommended = function (placement) {
```

Comment:
Why the need for `prototype` here? None of the other models needed it.

Reply:
It adds the method onto the Model instances. So instead of `Recommendation.findRecommended()`, It's (new Recommendation().findRecommended()`
This gives the function access to the `this` object which is the instance of the recommendation. We use it to find by similar category.